src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -107,6 +107,12 @@ public string ClientVersion
 
         public bool RetryOnServerErrors { get; set; } = true;
 
+        public bool UseManagedIdentity { get; internal set; }

```

Comment:
Maybe prefix these with ManagedIdentity or MI or smth like this to tell them apart from other variables.